ui/src/kapacitor/components/config/RedactedInput.tsx

```diff
@@ -31,7 +37,7 @@ class RedactedInput extends Component {
             id={id}
             type="hidden"
             ref={refFunc}
-            defaultValue={defaultValue}
+            defaultValue={defaultValue.toString()}
```

Comment:
if this is undefined or a boolean it seems like it'll convert to `'true'` or `'false'` or will fail if it's undefined, no?

Reply:
defaultValue in an input must be a string or array of strings.

Reply:
but then might we end up with an incorrect default value? such as the strings `true`, `false`, or `undefined`? seems like the guard should be higher up in the chain to prevent incorrect values from reaching this point